crates/core/tedge/tests/main.rs

```diff
@@ -47,36 +47,36 @@ mod tests {
         let home_dir = tempdir.path().to_str().unwrap();
 
         let mut get_device_id_cmd =
-            tedge_command_with_test_home(["--config-dir", home_dir, "config", "get", "device.id"])?;
+            tedge_command_with_test_home(["config", "get", "device.id", "--config-dir", home_dir])?;
```

Comment:
I have for now no nice solution to accept the old syntax with `--config-dir` set before the subcommand.